deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -397,6 +397,28 @@ cephFileSystems:
         # in hyperconverged settings where the volume is mounted on the same node as the osds.
         csi.storage.k8s.io/fstype: ext4
 
+cephFileSystemVolumeSnapshotClass:
+  enabled: true
+  name: ceph-filesystem-vsc
```

I'd suggest not including the vsc suffix in the name since it's already in the type.
```suggestion
  name: ceph-filesystem
```